workspace/utilities/admin-tools/admin-tools.css

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+.admin-tools{
+	position: fixed;
+	top: 10px;
+	left: 10px;
+	background-color: #2f77eb;
+	color: white;
+	font-size: 14px;
+	font-family: 'Roboto', sans-serif;
+	z-index: 99999;
+}
+
+.admin-tools a{
+	color: white;
+	text-decoration: none;
+}
+
+.admin-tools .notifier{
+	position: fixed;
+	top: 10px;
+	left: 10px;
+	width: 10px;
+	height: 10px;
+	margin: 5px 0 0 7px;
+	border-radius: 50%;
+	z-index: 99999;
+	box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+}
+
+.admin-tools .widgets{
+	position: absolute;
+	top:0;
+	opacity: 0;
+	display: none;
+	background-color: #1976D2;
+	border-radius: 8px;
+	box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+}
+
+.admin-tools:hover .widgets{
+	display: block;
+	animation: fadein 0.2s forwards;
+}
+
+.admin-tools .widget{
+	height: 24px;
+	padding: 0 15px 0 5px;
+}
+
+.admin-tools .widget-svg{
+	width: 15px;
+	margin-right: 5px;
+}
+
+@keyframes fadein {
+	from {transform: translateY(-5px);opacity: 0;}
+	to {transform: translateY(0px);opacity: 1;}
+}
```

Comment:
I would inline the shit out of this, no ?

Reply:
it might become inline in the future. But I want the tools to live their life a bit before committing to inlinyness.

Reply:
ok and @PascalPiche seems fine with it